crates/paths/src/paths.rs

```diff
@@ -83,6 +89,21 @@ pub fn config_dir() -> &'static PathBuf {
     })
 }
 
+/// Returns the path to the global (all users) config directory used by Zed.
+pub fn global_config_dir() -> &'static PathBuf {
+    GLOBAL_CONFIG_DIR.get_or_init(|| {
+        if cfg!(target_os = "windows") {
+            if let Ok(program_data) = std::env::var("ProgramData") {
+                PathBuf::from(program_data).join("Zed")
+            } else {
+                PathBuf::from("C:\\ProgramData\\Zed")
```

Comment:
It would be more robust to use [`FOLDERID_ProgramData`](https://microsoft.github.io/windows-docs-rs/doc/windows/Win32/UI/Shell/constant.FOLDERID_ProgramData.html) to get path of rogramData as the drive letter could be another letter.